v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/PipelineController.java

```diff
@@ -218,16 +237,6 @@ private static String generateSuffix(String shardId, String tableName) {
     return suffix;
   }
 
-  @VisibleForTesting
-  static SpannerConfig createSpannerConfig(SourceDbToSpannerOptions options) {
```

Comment:
This function was not referenced in the code. Removed as this was marked unused.